### PR TITLE
chore(stoneintg-1023): add IS SLO availability

### PR DESCRIPTION
**Pull request by jencull**
integration service availability metric
alert rule and alert test added

### CODE REVIEW

File: rhobs/alerting/data_plane/prometheus.availability_metric_integration_service_alerts.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        severity: critical
+        slo: "true"
+      annotations:
+        summary: >-
+          Integration Service Availability SLO Violation
+        description: >-
+          Integration Service GitHub App availability has dropped below 99% for more than 10 minutes on cluster {{ $labels.source_cluster }}.
+        alert_team_handle: <!subteam^S05M4AG8CJH>
+        team: integration
+        runbook_url: https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/integration-service/sre/integration_service_availability.md
```
Comment (yftacherzog) [inline rhobs/alerting/data_plane/prometheus.availability_metric_integration_service_alerts.yaml:28]: Can you add a newline at end of file?

**Review by yftacherzog — APPROVED** [review]
lgtm

File: test/promql/tests/data_plane/availability_metric_integration_service.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  # Test 1: Availability drops below 99% for more than 10 minutes on cluster01, triggering an alert.
+  - interval: 1m
+    input_series:
+      - series: 'redhat_appstudio_integrationservice_global_github_app_available{source_cluster="cluster01"}'
+        values: '1x50 0x70'  # 50 minutes of uptime, followed by 70 minutes of downtime
+      - series: 'redhat_appstudio_integrationservice_global_github_app_available{source_cluster="cluster02"}'
+        values: '1x120'  # Cluster02 remains up for the entire period.
+
+    alert_rule_test:
+      - eval_time: 60m
```
Comment (yftacherzog) [inline test/promql/tests/data_plane/availability_metric_integration_service.yaml:16]: I believe yaml-lint complains about trailing spaces on this line
```suggestion
      - eval_time: 60m
```

File: rhobs/alerting/data_plane/prometheus.availability_metric_integration_service_alerts.yaml (hunk 1 above)
Comment (jencull) [reply rhobs/alerting/data_plane/prometheus.availability_metric_integration_service_alerts.yaml:28]: yes, thank you

**Comment by jencull** [comment]
@redhat-appstudio/konflux-sre  for review, thanks.

**Review by yftacherzog — APPROVED** [review]
lgtm

**Comment by yftacherzog** [comment]
> @redhat-appstudio/konflux-sre for review, thanks.

@jencull Are you expecting more reviews? Ready to be merged?

**Comment by jencull** [comment]
> > @redhat-appstudio/konflux-sre for review, thanks.
> 
> @jencull Are you expecting more reviews? Ready to be merged?

ready to be merged, thanks @yftacherzog

**Pull request merged**